worlds/witness/options.py

```diff
@@ -123,26 +143,33 @@ class EnvironmentalPuzzlesDifficulty(Choice):
 class ObeliskKeys(DefaultOnToggle):

```

Comment:
I can't actually comment on the EP Difficulty section but you could change `On "X"` to `If set to "X"`